### DISCUSSION TITLE
请教下应用场景是什么？怎么用打开后一头雾水

### DISCUSSION THREAD

**Discussion by ishowman** [CaoMeiYouRen/rss-impact-server#470] [Q&A] [answered]
能分享下吗？

**Comment by CaoMeiYouRen** [comment]
有关使用文档的优化还在进行中，请等待一段时间。
可以提出一些更加具体的问题，如果比较常见的话，会考虑收录在文档中。
https://github.com/CaoMeiYouRen/rss-impact-server/issues/437

**Comment by CaoMeiYouRen** [comment, marked as answer]
请参考[使用文档](https://rss-docs.cmyr.dev/docs/usage)
